integration_tests/src/tests/multisig_test.rs

```diff
@@ -77,9 +77,7 @@ pub fn proposal_hash_test(v: &dyn VM) {
         approved: vec![alice],
         params: RawBytes::default(),
     };
-    let correct_hash =
-        compute_proposal_hash(&correct_tx, &MockRuntime::new(&DynBlockstore::wrap(v.blockstore())))
-            .unwrap();
+    let correct_hash = compute_proposal_hash(&correct_tx, &MockRuntime::new()).unwrap();
```

Comment:
Wait, actually, shouldn't this use `v.primitives()` as well?

Reply:
This is done.